src/types/json.h

```diff
@@ -39,6 +39,11 @@
 constexpr ssize_t NOT_FOUND_INDEX = -1;
 constexpr ssize_t NOT_ARRAY = -2;
 
+enum class NumOpEnum : uint8_t {
+  Incr = 1,
+  Mul = 2,
+};
```

Comment:
I think it'd be better to put this enum inside JsonValue to avoid name conflict.

Reply:
is that put the definition in the JsonValue?

```
struct JsonValue {

  enum class NumOpEnum : uint8_t {
    Incr = 1,
    Mul = 2,
  };
}
```

Reply:
Exactly.